### ISSUE TITLE
Alternate authentication scheme (SQRL)

### ISSUE THREAD

**Issue by measlytwerp** [rustodon/rustodon#122]
It would be nice to support [Secure Quick Reliable Logins](https://www.grc.com/sqrl/sqrl.htm).

Sorry for the short issue, don't have much time just at the minute.

**Comment by barzamin** [comment]
yeah, SQRL looks really cool! wondering if there are any, eg, nice mobile client apps atm?
